core/heap.go

```diff
@@ -0,0 +1,62 @@
+package core
+
+// transfer2go/utils - Go utilities for transfer2go
+//
+// Copyright (c) 2017 - Valentin Kuznetsov <vkuznet@gmail.com>
```

Comment:
Rishi, could you please from now on add "Author: " line instead of Copyright. And, add your name. I modified all code to follow new convention. Please pull out changes from the tree.